examples/nuxt/components/HelloWorld.vue.css.ts

```diff
@@ -0,0 +1,5 @@
+import { style } from '@vanilla-extract/css'
+
+export const externalStyle = style({
+  color: 'red'
+})
```

Comment:
This is as barebones as it gets. How do you feel about showing at least a few more of the basic features, like themes, vars, variants and maybe dynamic variables?